packages/host/app/components/matrix/room.gts

```diff
@@ -123,6 +155,11 @@ export default class Room extends Component<RoomArgs> {
           @disabled={{this.doChooseCard.isRunning}}
           {{on 'click' this.chooseCard}}
         >Choose Card</Button>
+        <Button
+          data-test-send-open-cards-btn
+          @loading={{this.doSendMessage.isRunning}}
+          {{on 'click' this.sendOpenCards}}
```

Comment:
might wanna also add `disabled={{this.doSendMessage.isRunning}}` so that the button is not clickable while the task is running